src/main/java/dev/ftb/extendedexchange/config/ConfigHelper.java

```diff
@@ -27,4 +27,6 @@ public static boolean isStoneTableWhitelisted(ItemStack stack) {
     public static BigInteger getEMCLinkMaxOutput() {
         return emcLinkMaxOutput;
     }
+
+    public static int getEMCLinkMaxStackSize() { return server().general.emcLinkMaxStackSize.get(); }
```

Comment:
Please be consistent with our code layout.

Reply:
Oh my bad! I had a messed up local repository and copied out the files through IntelliJ instead of through Windows Explorer so it didn't copy the formatting. Will push a commit to address this real quick! :D